ansible/templates/whisk.properties.j2

```diff
@@ -80,8 +80,8 @@ db.port={{ db_port }}
 db.username={{ db_username }}
 db.password={{ db_password }}
 db.prefix={{ db_prefix }}
-db.whisk.actions={{ db_prefix }}whisks
-db.whisk.auths={{ db_auth }}
+db.whisk.actions={{ db.whisk.actions }}
```

Comment:
> The names of the databases stay exactly the same.

If it's stays the same it's good. I missed this. I was rather referring on not introducing a breaking change.